upgrade

YahiaAbusaif opened the pull request.

YahiaAbusaif committed:
upgrade

Rezve commented:
So what features do you added. Please clarify. 

And change your comment from Arabic to English. I think that's way more people can understand your code better.

YahiaAbusaif committed:
update

add comments , add macro and pros so any upgrade at future will be more easy add feature change direction of the arrow

YahiaAbusaif commented:
I will add comments ,I added macro and pros so any upgrade at future will be more easy, fix some problem when I run it emulator on my PC, delete some unnecessary code and add feature change direction of the arrow

Rezve commented:
If you’ve made improvements to the game, please add the details to the description. You can also include your repository link in the README file.

Apologies for closing this PR, and thank you for your understanding.
